model_utils/managers.py

```diff
@@ -99,7 +99,10 @@ def _chain(self, **kwargs):
 
     def _clone(self, klass=None, setup=False, **kwargs):
         if django.VERSION >= (2, 0):
-            return super(InheritanceQuerySetMixin, self)._clone()
+            qs = super(InheritanceQuerySetMixin, self)._clone()
+            if hasattr(self, 'subclasses'):
+                qs.subclasses = self.subclasses
```

Comment:
Is there a reason to not move the copying logic from lines 107-109 above the version check? In particular, this change doesn't account for `_annotated`.

Reply:
Problem is that in django2 and above queryset does not allow `kwargs` in the clone function. 
Therefore the copying logic is a bit different. 
I created a new pull-request #335 to resolve this also for the `_annotated` field.